substrate/client/network/benches/notifications_protocol.rs

```diff
@@ -156,12 +153,19 @@ where
 				tokio::select! {
 					Some(event) = notification_service1.next_event() => {
 						if let NotificationEvent::NotificationStreamOpened { .. } = event {
-							break;
+							// Send a 32MB notification to preheat the network
+							notification_service1.send_async_notification(&peer_id2, vec![0; 2usize.pow(25)]).await.unwrap();
```

Comment:
We need to do a similar thing with requests: before doing any actual measurements request 32MB to reach the maximum TCP speed. Might be needed to modify the default max response size, as AFAIR it is < 32MB by default.